apisix/core/request.lua

```diff
@@ -278,6 +278,15 @@ function _M.get_path(ctx)
 end
 
 
+function _M.get_uri(ctx)
```

Comment:
So I should just use ctx.var.request_uri instead of wrapping it? Can I merge the functionality of `get_path` into `get_uri` and add a `with_args` parameter indicating whether or not I need to carry parameters.

Reply:
Don't make thing complex!
The name of request_uri and uri already show the difference. There is no need to wrap them into a function which is rarely used. Premature optimization is the source of evil.

Reply:
We can remove the `get_path` in the next PR. Better to get rid of it before the release, so that people won't ask why we use `get_path` in one place but use `ctx.var.uri` in the most elsewhere.

Reply:
removed